deploy/charts/openebs-monitoring/rules/lvm-LocalPV/lvm-localpv-pool-alerts.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - name: lvm-pool
+    rules:
+      # Alert for volume groups when the PV forming the VG is missing 
+      - alert: LVMVolumeGroupMissingPhysicalVolume
+        annotations:
+          componentType: "volume group"
+          vgName: "{{ $labels.name }}"
+          summary: 'LVM volume group ''{{ $labels.name }}'' is missing the underlying physical volume.'
+          description: |-
+            LVM volume group '{{ $labels.name }}' on node '{{ $labels.instance }}' is missing {{ $value }} underlying physical volumes used by it
```

Comment:
volume(s) and end the sentence there with full-stop. "used by it" is not needed.

Reply:
Is it possible to print the names of the missing PVs?

Reply:
Not sure if that is possible as we will have to write different queries in the summary to get the names but then only one row will be processed from the return list(say 3 rows). Also the names are present as labels and getting only the label and not the value of the query is not possible as far as I know.